libavfilter/dnn/dnn_backend_native_layer_avgpool.h

```diff
@@ -33,7 +33,36 @@ typedef struct AvgPoolParams{
     DNNPaddingParam padding_method;
 } AvgPoolParams;
 
+/**
+ * @brief Load Average Pooling Layer.
+ *
+ * It assigns the layer parameters to the hyperparameters 
+ * like strides, padding method, and kernel size after 
+ * parsing from the model file context.
+ * 
+ * Possible values for the padding method are SAME and VALID.
```

Comment:
maybe no need to specify the value?

Reply:
I thought that the padding method should be correctly selected to load the layer. But yes, since it is parsed from the model file context, maybe we don't need to specify its value. 🤔

Reply:
Should we specify the values for the padding method in the typedef?